lib/ui/common/widgets/ActionSheet.dart

```diff
@@ -0,0 +1,193 @@
+import 'package:farmsmart_flutter/ui/common/ActionSheetLargeRoundedButtonStyle.dart';
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:flutter/material.dart';
+
+class ActionListItemViewModel {
+  String title;
+  String icon;
+  String checkBoxIcon;
+  bool isHighlighted;
+  Function action;
+
+  ActionListItemViewModel(this.title, this.action, this.isHighlighted, {this.icon, this.checkBoxIcon});
+}
+
+class ActionSheetViewModel {
+  List<ActionListItemViewModel> actions;
+  String buttonTitle;
+
+  ActionSheetViewModel(this.actions, this.buttonTitle);
+}
+
+abstract class ActionSheetStyle {
+  final Color cornersColor;
+  final Color backgroundColor;
+  final Color dropLineColor;
+  final Color cardBackgroundColor;
+
+  final TextStyle mainTextStyle;
+  final TextStyle highlightTextStyle;
```

Comment:
re above comment: rename to destructiveTextStyle

Reply:
Done